tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -0,0 +1,258 @@
+import csv
+import datetime
+import json
+import zoneinfo
+
+import pytest
+
+from sds_data_manager.lambda_images.instruments.batch_starter import (
+    all_dependency_present,
+    get_filename_from_event,
+    get_process_details,
+    prepare_data,
+    query_dependencies,
+    query_dependents,
+    remove_ingested,
+)
+
+
+@pytest.fixture(scope="session")
+def mock_event():
```

Comment:
Should we be using moto here to mock the s3 event instead?
https://docs.getmoto.org/en/latest/docs/services/events.html

Reply:
I couldn't figure out a way to generate an event payload using moto. I basically just wanted to test if we could get the filename from a event from an object dropped in the s3 bucket.